Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/DriverFee.hs

```diff
@@ -209,7 +209,7 @@ updateRegisterationFeeStatusByDriverId status (Id driverId) = do
 updateCollectedPaymentStatus :: MonadFlow m => DriverFeeStatus -> Maybe Text -> UTCTime -> Id DriverFee -> m ()
 updateCollectedPaymentStatus status collectorId now (Id driverFeeId) = do
   updateOneWithKV
-    [Se.Set BeamDF.status status, Se.Set BeamDF.updatedAt now, Se.Set BeamDF.collectedBy collectorId]
+    [Se.Set BeamDF.status status, Se.Set BeamDF.updatedAt now, Se.Set BeamDF.collectedBy collectorId, Se.Set BeamDF.collectedAt (Just now)]
```

Comment:
This query is called even if the status is `EXEMEPTED`. Can you add a check to only update in case of `COLLECTED_CASH`?